### ISSUE TITLE
Toast example demo code error

### ISSUE THREAD

**Issue by TrendAlan** [trendmicro-frontend/tonic-ui#283]
The [example code](https://trendmicro-frontend.github.io/styled-ui/toast) for useToast will report a javascript error

```
<ThemeProvider theme={theme}>
  <ColorModeProvider value={colorMode}>
     <ColorStyleProvider>
        {render({ id, onClose, position, duration })}
      </ColorStyleProvider>
  </ColorModeProvider>
</ThemeProvider>
```

console error:
_Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports._

But after remove **ColorStyleProvider** it works
```
<ThemeProvider theme={theme}>
  <ColorModeProvider value={colorMode}>
    {render({ id, onClose, position, duration })}
  </ColorModeProvider>
</ThemeProvider>
```

**Comment by cheton** [comment]
@TrendAlan 
May I know your installed version of `@trendmicro/react-styled-ui`?

The `ColorStyleProvider` is available since `v0.13.0`:

https://github.com/trendmicro-frontend/styled-ui/releases/tag/%40trendmicro%2Freact-styled-ui%400.13.0


It's also recommended to upgrade to v0.13.1 with the latest bug fix.
https://github.com/trendmicro-frontend/styled-ui/releases/tag/%40trendmicro%2Freact-styled-ui%400.13.1
* 🐛 fix: `uncaught Error: [ThemeProvider] Please make your theme prop a plain object` (#280)
* 🐛 fix: fix an issue that cannot pass the z-index value to `DrawerContent` and `ModalContent` components (#281)

**Comment by TrendAlan** [comment]
@cheton 
My mistake, I was still using version **v0.12.1**, I will upgrade to the latest version and try again.
Thank you
